### DISCUSSION TITLE
about horiz_time_volume function

### DISCUSSION THREAD

**Discussion by Special-Boo** [highfestiva/finplot#473] [Q&A]
![image](https://github.com/highfestiva/finplot/assets/95997280/8209fd75-a5f0-4d23-a39b-a75e83e9fd53)

Is it possible to adjust the alpha value of the bars in horiz_time_volume? I would like to adjust the alpha value for visibility. -- (Thank you for making finPlot! It’s really useful.)

**Comment by highfestiva** [comment]
Yes, finplot support RGBA colors, where the last two nibbles represent alpha value (transparency). Do like so:

```python
htv_plot = fplt.horiz_time_volume(...)
htv_plot.colors['neutral_frame'] = '#00000040' # black with about 25% opacity
htv_plot.colors['neutral_body'] = '#ff000020' # red with about 12.5% opacity
```

finplot also supports three nibble colors RGB, like `#f00` and four-nibble RGBA colors like `#0ff5`. Thanks and have fun!

**Reply by Special-Boo** [reply]
Thanks!!
